services/tenant-ui/frontend/src/store/contactsStore.ts

```diff
@@ -59,7 +59,44 @@ export const useContactsStore = defineStore('contacts', () => {
     return invitation_data;
   }
 
-  return { contacts, selectedContact, loading, error, listContacts, createInvitation };
+  async function acceptInvitation(inviteUrl: string, alias: string) {
+    console.log('> contactsStore.acceptInvitation');
+    error.value = null;
+    loading.value = true;
+
+    let accepted_data = null;
+    // need the await here since the returned invitation_data is not one of our stored refs...
+    await tenantApi
+      .postHttp('/tenant/v1/contacts/receive-invitation', { alias: alias, invitation_url: inviteUrl })
+      .then((res) => {
+        console.log(res);
+        // don't grab the item, there are other parts of the response data we need (invitation, invitation url)
+        accepted_data = res.data;
+        console.log(accepted_data);
+      })
+      .then(() => {
+        // do we want to automatically reload? or have the caller of this to load?
+        console.log('invitation accepted. the store calls load automatically, but do we want this done "manually"?');
+        listContacts();
```

Comment:
Reloading after as with the create contact part in the store itself. I think this'll be the pattern to stick with but can see as we go.

Reply:
totally... we will see what patterns emerge.  contacts is probably the safest to "auto" refresh as it will be in pick lists everywhere.